pkg/addon/policyframework/manifests/managedclusterchart/templates/cluster_role_binding_crd_listwatch.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
I'm 70% sure this will need to be a ClusterRoleBinding (but not 100% sure). Based on and example in https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples
> because a Node is cluster-scoped, this must be in a ClusterRole bound with a ClusterRoleBinding to be effective

My understanding is, you can use a RoleBinding to bind to a ClusterRole, but it wouldn't give permissions for cluster-wide resources.